### ISSUE TITLE
Darkmode rendering failure in Windows 11

### ISSUE THREAD

**Issue by shriprem** [shriprem/NPP_Plugin_Darkmode#2]
The backgrounds of plugin UIs are not rendering while in darkmode in Windows 11.

### GotoLineCol dock panel
<img width="274" alt="GTCPanel" src="https://user-images.githubusercontent.com/45252729/146485823-65e2513f-8698-408b-a2db-3af6a67cce21.png">

### GotoLineCol Preferences dialog
<img width="390" alt="GTCPreferences" src="https://user-images.githubusercontent.com/45252729/146485824-7498dd46-8680-4b61-a045-ae2892f293c6.png">

### FWDataViz dock panel
<img width="275" alt="FWDVPanel" src="https://user-images.githubusercontent.com/45252729/146486087-7f93691b-6a89-4f29-aff4-92e410c02bf7.png">

### FWDataViz Themes dialog
<img width="698" alt="FWDVThemesDialog1" src="https://user-images.githubusercontent.com/45252729/146486240-851d020d-e3c6-4339-9617-81b45f29df09.png">

**Comment by shriprem** [comment]
Fix involved syncing code changes for NPPDarkMode.cpp, DarkMode.cpp & IatHook.h from the Notepad++ repo. The .lib & .dll files will be available in [Release v1.1.0.0](https://github.com/shriprem/NPP_Plugin_Darkmode/releases/tag/v1.1.0.0)

## Corresponding screenshots after the fix

### GotoLineCol dock panel
<img width="279" alt="GTCPanel2" src="https://user-images.githubusercontent.com/45252729/146488810-4dbfd0a0-c880-4459-bd15-af9dd3ddf002.png">

### GotoLineCol Preferences dialog
<img width="390" alt="GTCPreferences2" src="https://user-images.githubusercontent.com/45252729/146488821-28d5925c-a077-44ca-936d-b780ded6990f.png">

### FWDataViz dock panel
<img width="283" alt="FWDVPanel2" src="https://user-images.githubusercontent.com/45252729/146488868-0a8d2fdd-6195-470a-9a05-653c42753b79.png">

### FWDataViz Themes dialog
<img width="698" alt="FWDVThemesDialog3" src="https://user-images.githubusercontent.com/45252729/146488895-a14215d1-a935-4243-89d4-e15ed93efb90.png">

**Comment by shriprem** [comment]
The only critical change that actually mattered for the fix was on [line# 1231 in NppDarkMode.cpp](https://github.com/shriprem/NPP_Plugin_Darkmode/blob/ec5fe307e6c680b1dcf7792ed2f10006b2b8d7bd/NPP_Plugin_Darkmode/NppDarkMode.cpp#L1231).

In the [commit](https://github.com/shriprem/NPP_Plugin_Darkmode/commit/ec5fe307e6c680b1dcf7792ed2f10006b2b8d7bd), it is actually coded as:
```C++
::EnableThemeDialogTexture(hwndParent, theme && !NppDarkMode::isEnabled() ? ETDT_ENABLETAB : ETDT_DISABLE);
```

However, it could have been much simpler and as effective if it was just coded as:
```C++
EnableThemeDialogTexture(hwndParent, ETDT_DISABLE);
```
